lib/logstash/outputs/elasticsearch_java.rb

```diff
@@ -238,14 +228,11 @@ class LogStash::Outputs::ElasticSearch < LogStash::Outputs::Base
   # in situations where you cannot permit connections outbound from the
   # Elasticsearch cluster to this Logstash server.
   #
-  # The 'http' protocol will use the Elasticsearch REST/HTTP interface to talk
-  # to elasticsearch.
-  #
   # All protocols will use bulk requests when talking to Elasticsearch.
   #
   # The default `protocol` setting under java/jruby is "node". The default
```

Comment:
Old comment about what is 'default' per-platform, but not relevant anymore (Yay!) so let's delete these two lines.